resources/js/features/achievements/components/AchievementBreadcrumbs/AchievementBreadcrumbs.tsx

```diff
@@ -34,7 +35,9 @@ export const AchievementBreadcrumbs: FC<AchievementBreadcrumbsProps> = ({
       <BaseBreadcrumb>
         <BaseBreadcrumbList>
           <BaseBreadcrumbItem>
-            <BaseBreadcrumbLink href="/gameList.php">All Games</BaseBreadcrumbLink>
+            <BaseBreadcrumbLink asChild>
+              <Link href={route('game.index')}>All Games</Link>
+            </BaseBreadcrumbLink>
```

Comment:
`<Link />` does client-side routing to the next page, and works as long as that page is also an Inertia.js page. This results in faster page transitions.